### PR TITLE
Homepage improvements

### PR DESCRIPTION
**Pull request by ritika-b**
- updates homepage organization to be more uniform
- Updates titles for uniformity
- add TOC [headline is Topics]
- Remove overview section
- create uniform content structure
- change "What's Next" heading

### CODE REVIEW

File: docs/Using Price Feeds/using-chainlink-reference-contracts.md [hunk 1]
```diff
@@ -2,34 +2,40 @@
...
-## Connect your contracts to the outside world
-
 Chainlink Data Feeds are the quickest way to connect your smart contracts to the real-world data such as asset prices. One use for data feeds is to retrieve the latest pricing data of an asset in a single call and use that data either on-chain in a smart contract or off-chain in another application of your choice.
 
 If you already have a project started and would like to integrate Chainlink, you can [add Chainlink to your existing project](../create-a-chainlinked-project/#install-into-existing-projects) by using the [`chainlink` NPM package](https://www.npmjs.com/package/@chainlink/contracts).
 
 See the [Data Feeds Contract Addresses](/docs/reference-contracts/) page for a list of networks and proxy addresses.  
 
-For important updates regarding the use of Chainlink Price Feeds, users should join the official Chainlink Discord and subscribe to the data-feeds-user-notifications channel: https://discord.gg/Dqy5N9UbsR
+**Topics**
```
Comment (dwightjl) [inline docs/Using Price Feeds/using-chainlink-reference-contracts.md:20]: I like this change. Shall we switch all of our `**Table of contents**` entries to say `**Topics**` instead?

**Review by dwightjl — CHANGES_REQUESTED** [review]

File: docs/Using Price Feeds/using-chainlink-reference-contracts.md (hunk 1 above)
Comment (ritika-b) [reply docs/Using Price Feeds/using-chainlink-reference-contracts.md:24]: I think if we agree to change it on this page, we should carry it out to all pages for consistency.

File: docs/Using Price Feeds/using-chainlink-reference-contracts.md [hunk 2]
```diff
@@ -2,28 +2,38 @@
...
+- [Retrieve the latest asset prices](#retrieve-the-latest-asset-prices) 
+- [Components of a data feed](#components-of-a-data-feed)
+- [Reading proxy and aggregator Configurations](#reading-proxy-and-aggregator-configurations)
+- [Components of an aggregator](#components-of-an-aggregator)
+- [Updates to proxy and aggregator contracts](#updates-to-proxy-and-aggregator-contracts)
+- [Monitoring data feeds](#monitoring-data-feeds)
+- [Check the latest answer against reasonable limits](#check-the-latest-answer-against-reasonable-limits)
+- [Check the timestamp of the latest answer](#check-the-timestamp-of-the-latest-answer)
```
Comment (dwightjl) [inline docs/Using Price Feeds/using-chainlink-reference-contracts.md:32]:
```suggestion
  - [Check the latest answer against reasonable limits](#check-the-latest-answer-against-reasonable-limits)
  - [Check the timestamp of the latest answer](#check-the-timestamp-of-the-latest-answer)
```

**Review by dwightjl — CHANGES_REQUESTED** [review]

File: docs/Using Price Feeds/using-chainlink-reference-contracts.md (hunk 1 above)
Comment (ritika-b) [reply docs/Using Price Feeds/using-chainlink-reference-contracts.md:24]: Resolved

**Review by dwightjl — APPROVED** [review]

**Pull request merged**